File: Character.cs
using System;
namespace SuperBasicRPG
{
    class Character : EntityLivingModel
    {
        public Character()
        {
            // WHERE/HOW DO I INITIALIZE THE DATABASE???
            // Seperate file called database.cs where all that information takes place.
            // Once I get the database initialization completed I wont have to worry about it, Set it and Forget it.
            Console.WriteLine("Checking Database...");
        }
        public bool UpdateName(string enteredName)
        {
            bool updateNameSuccessCheck = false;
            this.name = enteredName;
            return updateNameSuccessCheck;
        }
        public bool UpdateRace(string enteredRace)
        {
            bool updateRaceSuccessCheck;
            this.race = enteredRace;
            if (GetRace() == enteredRace)
            {
                updateRaceSuccessCheck = true;
            }
            else  // LET ME KNOW IF THIS IS COMPLETELY UNNECESSARY... or if I did it correctly (which is doubtful)
            {
                updateRaceSuccessCheck = false;
            }
            return updateRaceSuccessCheck;
        }/*
        public bool UpdateClass(string enteredClass)
        {
            this.characterClass = enteredClass;
            bool updateClassSuccessCheck = true;   // This section is obviously wrong but I dont know how to fix it yet. 
            return updateClassSuccessCheck;
        }

        internal string GetCharacterClass()
        {
            return this.characterClass; // I need to have the 'class choice option' functional in order to write to the DB
        }
        internal int GetWeapon()
        {
            return this.weaponInfo; // I need to have the weapon system semi-functional in order to write to the DB
        }
        internal int GetTool()
        {
            return this.toolInfo; // I need to have the tool system semi-functional in order to write to the DB
        }
        */
        internal string GetName()
        {
            return this.name;
        }
        internal string GetRace()
        {
            return this.race;
        }
        internal int GetLevel()
        {
            return this.level;
        }
        internal int GetCurrentXP()
        {
            return this.experience;
        }
        internal int GetHealthCurrent()
        {
            return this.health_Current;
        }
        internal int GetHealthMax()
        {
            return this.health_Max;
        }
        internal int GetManaCurrent()
        {
            return this.mana_Current;
        }
        internal int GetManaMax()
        {
            return this.mana_Max;
        }

    }
}

File: py/character.py
class Character: #EntityLivingModel
    def __init__(self):
        # WHERE/HOW DO I INITIALIZE THE DATABASE???
        # Separate file called database.py where all that information takes place.
        # Once I get the database initialization completed I won't have to worry about it, Set it and Forget it.
        print("Checking Database...")
    
    def update_name(self, entered_name):
        self.name = entered_name
        return True

    def update_race(self, entered_race):
        self.race = entered_race
        return self.get_race() == entered_race

    def update_class(self, entered_class):
        self.character_class = entered_class
        return True  # This section is obviously wrong but I don't know how to fix it yet.
    
    def get_character_class(self):
        return self.character_class  # I need to have the 'class choice option' functional in order to write to the DB

    def get_weapon(self):
        return self.weapon_info  # I need to have the weapon system semi-functional in order to write to the DB

    def get_tool(self):
        return self.tool_info  # I need to have the tool system semi-functional in order to write to the DB

    def get_name(self):
        return self.name

    def get_race(self):
        return self.race

    def get_level(self):
        return self.level

    def get_current_xp(self):
        return self.experience

    def get_health_current(self):
        return self.health_current

    def get_health_max(self):
        return self.health_max

    def get_mana_current(self):
        return self.mana_current

    def get_mana_max(self):
        return self.mana_max